spark/v3.2/spark/src/main/java/com/netease/arctic/spark/table/ArcticSparkTable.java

```diff
@@ -66,39 +66,25 @@ public class ArcticSparkTable
           TableCapability.OVERWRITE_DYNAMIC);
 
   private final ArcticTable arcticTable;
-  private final StructType requestedSchema;
-  private final boolean refreshEagerly;
+  private final String sparkCatalogName;
   private StructType lazyTableSchema = null;
   private SparkSession lazySpark = null;
   private final ArcticCatalog catalog;
 
-  public static Table ofArcticTable(ArcticTable table, ArcticCatalog catalog) {
+  public static Table ofArcticTable(
+      ArcticTable table, ArcticCatalog catalog, String sparkCatalogName) {
     if (table.isUnkeyedTable()) {
       if (!(table instanceof SupportHive)) {
-        return new ArcticIcebergSparkTable(table.asUnkeyedTable(), false);
+        return new ArcticIcebergSparkTable(table.asUnkeyedTable(), false, sparkCatalogName);
       }
     }
-    return new ArcticSparkTable(table, false, catalog);
+    return new ArcticSparkTable(table, sparkCatalogName, catalog);
   }
 
-  public ArcticSparkTable(ArcticTable arcticTable, boolean refreshEagerly, ArcticCatalog catalog) {
-    this(arcticTable, null, refreshEagerly, catalog);
-  }
-
-  public ArcticSparkTable(
-      ArcticTable arcticTable,
-      StructType requestedSchema,
-      boolean refreshEagerly,
-      ArcticCatalog catalog) {
+  public ArcticSparkTable(ArcticTable arcticTable, String sparkCatalogName, ArcticCatalog catalog) {
```

Comment:
The order of parameters for this constructor is different from that in Spark 3.1 and 3.3.